src/static/libs/js/mediaPlayer.js

```diff
@@ -1,299 +1,30 @@
-/*eslint-disable*/
-'use strict';
+import { calcDuration } from './utility.js';
+import { convertSecToHHMMSS } from './utility.js';
```

Comment:
I'll do that. I thought it might get horizontally too long if I do that, so I wasn't sure if it was an improvement.